Sources/PrimerSDK/Classes/Core/Primer/Primer.swift

```diff
@@ -175,6 +176,85 @@ public class Primer {
             }
         }
     }
+    
+    public func showUniversalCheckout(on viewController: UIViewController, clientToken: String? = nil) {
+        Primer.shared.flow = .default
+        presentingViewController = viewController
+        
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self else { return }
+            self.root = RootViewController()
+            guard let root = self.root else { return }
+            let router: RouterDelegate = DependencyContainer.resolve()
+            router.setRoot(root)
+            viewController.present(root, animated: true)
+        }
+    }
+    
+    public func showVaultManager(on viewController: UIViewController, clientToken: String? = nil) {
```

Comment:
Doesn't seem right that `clientToken` is not used in the function 🤔 
I think we should just remove it for now.

Reply:
Fixed. Bear in mind that this can be used in two ways:

1. `.showVaultManager(on: vc)`
2. `.showVaultManager(on: vc, clientToken: "a_token")`

The second one will avoid a new request for token.